### DISCUSSION TITLE
Whether it can support pinyin matching Chinese search

### DISCUSSION THREAD

**Discussion by LinusChen-yf** [Ulauncher/Ulauncher#983] [Ideas]
For example：
search "微信" App, able to use pinyin "wexing" search.
Hope to make it happen in the future, thanks a lot.

**Comment by friday** [comment]
I'm not very knowledgeable in this topic, but if you want to support being able to input latinized characters and get matches for their equivalent in the language used by the local environment, I think this also should apply for Cyrillic for example. If we can support this without adding heavy dependencies or slowing down search I think it's a nice addition.

It won't be something I plan to work on, so it would have to be a user submission.

**Reply by LinusChen-yf** [reply]
Thank you for your answer.
I am not very familiar with this project and Python at present, and I hope to contribute to Chinese search for the project in the future.
